arangod/Aql/ConstrainedSortExecutor.h

```diff
@@ -93,7 +93,7 @@ class ConstrainedSortExecutor {
   Fetcher& _fetcher;
   ExecutionState _state;
   size_t _returnNext;
-  std::vector<uint32_t> _rows;
+  std::vector<size_t> _rows;
```

Comment:
How many entries do we expect to have inside `_rows`?
Effectively doubling the size of each entry will obviously increase memory usage and will reduce cache karma.
If we are talking about few elements only, fine. But I am not sure. @danielhlarkin ?

Reply:
So this limits our queries to only having 4B rows at the intermediate step prior to limit. It doesn't seem wise to me. The standard sort executor uses `size_t` for similar purposes as well.